opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java

```diff
@@ -16,11 +16,7 @@
 import io.opentracing.Span;
 import io.opentracing.SpanContext;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Good catch, must've been a side-effect from my IntelliJ settings, I'll correct this!